RadarSDK/RadarUser.m

```diff
@@ -369,12 +376,21 @@ - (NSDictionary *)dictionaryValue {
     NSArray *topChainsArr = [RadarChain arrayForChains:self.topChains];
     [dict setValue:topChainsArr forKey:@"topChains"];
     [dict setValue:[Radar stringForLocationSource:self.source] forKey:@"source"];
-    NSDictionary *fraudDict = @{@"proxy": @(self.proxy)};
-    [dict setValue:fraudDict forKey:@"fraud"];
+    [dict setValue:[self.fraud dictionaryValue] forKey:@"fraud"];
     if (self.trip) {
         [dict setValue:[self.trip dictionaryValue] forKey:@"trip"];
     }
     return dict;
 }
 
+- (bool)asBool:(NSObject *)object {
```

Comment:
should this belong in a more generic class, like `RadarUtils`? seems like an abstract operation that we might want to use beyond just `RadarUser`

Reply:
It definitely should be on an extension of `NSObject` or something, but I'm trying to take baby steps here. :)

(This is off-topic, but these model classes are prime candidates for converting to `Codable` objects in Swift--it would eliminate almost all of the existing parsing code. Even better would be using [Swagger](https://swagger.io) to define the APIs, and then letting _it_ generate the client- and server-side model code.)

Reply:
📌

Reply:
+1, would be nice to centralize this when we refactor serialization/deserialization logic later.

Does it matter that we're using `bool` instead of `BOOL` here? We use `BOOL` elsewhere, although not so sure it matters in Swift.

Reply:
I'm gonna swap it to `BOOL` while releasing to be consistent, if I'm doing something dumb we can undo it later

Reply:
That's fine; I was just slipping into a Swift accent.